src/test/java/org/gridsuite/study/server/SingleLineDiagramTest.java

```diff
@@ -533,7 +533,7 @@ private StudyEntity insertDummyStudy(UUID networkUuid, UUID caseUuid) {
             .connectedComponentMode(LoadFlowParameters.ConnectedComponentMode.MAIN)
             .build();
         ShortCircuitParametersEntity defaultShortCircuitParametersEntity = ShortCircuitService.toEntity(ShortCircuitService.getDefaultShortCircuitParameters());
-        StudyEntity studyEntity = TestUtils.createDummyStudy(networkUuid, caseUuid, "", defaultLoadflowProvider, defaultLoadflowParametersEntity, defaultShortCircuitParametersEntity, null, null);
+        StudyEntity studyEntity = TestUtils.createDummyStudy(networkUuid, caseUuid, "", defaultLoadflowProvider, defaultLoadflowParametersEntity, defaultShortCircuitParametersEntity, null, null, null);
```

Comment:
why not passing a defaultSensitivityAnalysisParametersEntity instead of null ?

Reply:
done